.gitignore

```diff
@@ -9,9 +9,9 @@ node_modules/
 
 # Local development and debugging
 .scratch/
-.vscode
-**/.vscode/*
+#.vscode
+#**/.vscode/*

```

Comment:
I don't want these removed - if you need an exception for the root `.vscode/launch.json`, I'd add that.